apps/cookbook/src/app/showcase/page-showcase/page-showcase.component.ts

```diff
@@ -187,20 +187,31 @@ export class PageShowcaseComponent {
     },
   ];
 
-  public injectionTokenExample = `export const MY_BACK_BUTTON_OVERRIDE: PageBackButtonOverride = (routerOutlet, navCtrl, defaultHref) => {
+  public injectionTokenExample = `@Injectable({
```

Comment:
Should we add the import of `PageBackButtonOverride` and `PAGE_BACK_BUTTON_OVERRIDE` to the snippet, so it's clear where it comes from?

Reply:
Coming up!

Reply:
Added in d48827b8d12820b8653a49ede071ab5eb7901a65